include/rocksdb/utilities/transaction_db.h

```diff
@@ -158,6 +158,11 @@ struct DeadlockPath {
 
 class TransactionDB : public StackableDB {
  public:
+  using StackableDB::Write;
+  virtual Status Write(const WriteOptions& opts, WriteBatch* updates,
+                       bool skip_cc) {
+    return StackableDB::Write(opts, updates);
```

Comment:
Maybe a comment on what skip_cc does and why it's not passed in here?

Reply:
Certainly. I will also add a comment that the default behavior is to ignore the skip_cc hint so we would not have to implement it rightaway in all TransactionDBs such as OptimisticTransactionDB.